internal/evmreader/evmreader.go

```diff
@@ -33,7 +33,8 @@ type (
 
 // Interface for Input reading
 type InputSource interface {
-	// Wrapper for FilterInputAdded(), which is automatically generated by go-ethereum and cannot be used for testing
+	// Wrapper for FilterInputAdded(), which is automatically generated
```

Comment:
These format changes should be moved to 1a56a8c5b29913079bf141604c7d74b74e51efd6